tools/wptrunner/wptrunner/testloader.py

```diff
@@ -23,6 +24,8 @@
 manifest_update = None
 download_from_github = None
 
+TestsByType = Mapping[Tuple[str, str], List[wpttest.Test]]
```

Comment:
This is tests by subsuite and type, right? Not sure if calling it `TestsBySubsuiteType` is too awkward. Maybe we can add a comment to make it clear what it is?